PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -76,6 +78,7 @@ class PsfPowershellScriptRunner
         bool stopOnScriptError = false;
         std::filesystem::path currentDirectory;
         bool doesScriptExistInConfig = false;
+        LPPROC_THREAD_ATTRIBUTE_LIST attributeList;
     };
 
     ScriptInformation startingScriptInformation;

```

Comment:
What if you keep the attributeList in the ScriptInformation object, but have it be optional?

e.g. std::optional<MyProcThreadAttributeList> attributeList;

Reply:
It should not be optional.  Scripts will always need the attribute list.  I made the attribute list a member variable so I would not a redundant copy of it.

---
In reply to: [396725258](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/136#discussion_r396725258) [](ancestors = 396725258)